messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/ServerDefinition.java

```diff
@@ -109,6 +109,14 @@
             .addAccessConstraint(SensitiveTargetAccessConstraintDefinition.SECURITY_DOMAIN_REF)
             .addAccessConstraint(MessagingExtension.MESSAGING_SECURITY_SENSITIVE_TARGET)
             .build();
+    public static final SimpleAttributeDefinition ELYTRON_DOMAIN = create("elytron-domain", ModelType.STRING)
```

Comment:
I am not keen on this but maybe it is unavoidable as there is already a security-domain attribute.

Elsewhere we are trying to use a consistent set of names for references, sasl-authentication-factory, http-authentication-factory, ssl-context, and security-domain.  

Those four I expect administrators to use across the server.